src/clj/rems/db/entitlements.clj

```diff
@@ -62,30 +62,56 @@
           (log/warnf "Post failed: %s", response))
         (db/log-entitlement-post! {:target target :payload json-payload :status status})))))
 
+(defn- accepted-licenses? [application user-id]
+  (every? (or (get (:application/accepted-licenses application)
+                   user-id)
+              #{})
+          (map :license/id (:application/licenses application))))
```

Comment:
Not sure if I understand this correctly, but isn't this true also for an empty hash-set?

Reply:
First part is geting the accepted application licenses for a particular user, or empty set if there is no user there can be no accepted licenses.

Then the second part is that we check that every license in the application is acceted. i.e. in the set of accepted application licenses of the user..
```
(every? #{} [1 2 3]) => false
```